### PR TITLE
Site maintenance - Update Data JSON Working Example and fix language

### PR DESCRIPTION
**Pull request by Ricokola**
- Updating  data tables working example in data JSON.
- Fixing the page language selector in French by inserting the i18n variable in core.hml

### CODE REVIEW

File: components/wb-data-json/demo/data-fr.json [hunk 1]
```diff
@@ -19,8 +19,17 @@
     { "nom": "Item 4", "prop": "violet" }
   ],
   "bureaux": [
-    { "nom": "Jean Edmonds, Tour nord", "num": 300, "rue": "Slater", "ville": "Ottawa", "css": "bg-success", "homeUrl": "https://github.com/wet-boew" },
-    { "nom": "Place du portage, Phase I", "num": 50, "rue": "Victoria", "ville": "Gatineau", "css": "bg-danger", "homeUrl": "https://github.com/wet-boew" }
+{ "nom": "Nom de lieu exemple 1", "num": 100, "rue": "Slater", "ville": "Ottawa", "codePostal": "J8X 0I6", "css": "bg-success", "homeUrl": "https://github.com/wet-boew", "courriel": "info1@example.com" },
+{ "nom": "Nom de lieu exemple 2", "num": 65, "rue": "Victoria", "ville": "Gatineau", "codePostal": "J8S 0M6", "css": "bg-danger", "homeUrl": "https://github.com/wet-boew", "courriel": "info2@example.com" },
+{ "nom": "Nom de lieu exemple 3", "num": 87, "rue": "Slater", "ville": "Ottawa", "codePostal": "J9X 0I9", "css": "bg-success", "homeUrl": "https://github.com/wet-boew", "courriel": "info3@example.com" },
```
Comment (duboisp) [inline components/wb-data-json/demo/data-fr.json:24]:
```json
  "bureaux": [
    { "nom": "Jean Edmonds, Tour nord", "num": 300, "rue": "Slater", "ville": "Ottawa", "css": "bg-success", "homeUrl": "https://github.com/wet-boew" },
    { "nom": "Place du portage, Phase I", "num": 50, "rue": "Victoria", "ville": "Gatineau", "css": "bg-danger", "homeUrl": "https://github.com/wet-boew" }
],
"listAdresse": [
{ "nom": "Nom de lieu exemple 1", "num": 100, "rue": "Slater", "ville": "Ottawa", "codePostal": "J8X 0I6", "css": "bg-success", "homeUrl": "https://github.com/wet-boew", "courriel": "info1@example.com" },
{ "nom": "Nom de lieu exemple 2", "num": 65, "rue": "Victoria", "ville": "Gatineau", "codePostal": "J8S 0M6", "css": "bg-danger", "homeUrl": "https://github.com/wet-boew", "courriel": "info2@example.com" },

```

**Review by duboisp — CHANGES_REQUESTED** [review]
Add the JSON code, as a HTML snapshot, at line 44 in the template-en.html

**Comment by duboisp** [comment]
Pre-approved after the JSON sample is added into the working example page

**Comment by Ricokola** [comment]
@duboisp @GormFrank The template en and fr have been updated as requested.

File: components/wb-data-json/template-en.html [hunk 2]
```diff
@@ -45,6 +45,19 @@
 		{ &quot;name&quot;: &quot;Jean Edmonds, North Tower&quot;, &quot;num&quot;: 300, &quot;street&quot;: &quot;Slater&quot;, &quot;city&quot;: &quot;Ottawa&quot;, &quot;css&quot;: &quot;bg-success&quot;, &quot;homeUrl&quot;: &quot;https://github.com/wet-boew&quot; },
 		{ &quot;name&quot;: &quot;Place du portage, Phase I&quot;, &quot;num&quot;: 50, &quot;street&quot;: &quot;Victoria&quot;, &quot;city&quot;: &quot;Gatineau&quot;, &quot;css&quot;: &quot;bg-danger&quot;, &quot;homeUrl&quot;: &quot;https://github.com/wet-boew&quot; }
```
Comment (duboisp) [inline components/wb-data-json/template-en.html:46]: The name will need to be updated to match the content in the JSON file

**Review by duboisp — CHANGES_REQUESTED** [review]
Once the code sample match the JSON content, we will merge it.

File: components/wb-data-json/template-en.html (hunk 2 above)
Comment (Ricokola) [reply components/wb-data-json/template-en.html]: @duboisp The name have been updated in both English in French. This is ready for final review and approval.

**Review by duboisp — APPROVED** [review]

**Pull request merged**